scripts/simulator_base.js

```diff
@@ -749,7 +761,7 @@ var SIMULATOR = {};
 				}
 			}
 
-			return true;
+			return affected;
```

Comment:
Tanit has "Bolt 80 Every 2" and it will not retry regardless if it failed because of Invisibility.
e.g. it will always strike on turns 2, 6, 10, 14...
It does seem inconsistent with freeze and confuse that do retry on failure, but I guess we should keep `return true` here

Reply:
Same for Weaken.